### PR TITLE
BZ-1962202: adds restrictions to important message

### PR DESCRIPTION
**Pull request by JoeAldinger**
BZ-1962202: adds missing information regarding the hybrid networking configuration with OVN-Kubernetes  

Version(s):
4.7

Issue:
[BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1962202) 

Link to docs preview:
[Preview](https://joealdinger.github.io/openshift-docs/BZ-1962202/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html)

Additional information:
@anuragthehatter PTAL for QE ack

### CODE REVIEW

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 1]
```diff
@@ -98,8 +98,15 @@ IMPORTANT: You cannot change the service network address block during the migrat
 * Geneve (Generic Network Virtualization Encapsulation) overlay network port
 --
 +
+[IMPORTANT]
+====
+You must configure hybrid networking with OVN-Kubernetes during the installation of your cluster. You cannot switch to hybrid networking after the installation process.
+
+In addition, the hybrid  OVNKubernetes cluster is a requirement for Windows Machine Config Operator (WMCO)
```
Comment (abrennan89) [inline modules/nw-ovn-kubernetes-migration.adoc:105]: Should this also be "OVN-Kubernetes cluster" for consistency?

Also, re "for Windows Machine Config Operator (WMCO)", I'd say "for the Windows Machine Config Operator (WMCO)." (don't forget the period at the end of the sentence 😉 )

For Operators, I'd recommend adding an attribute for the name to save you from typing it out each time it's re-used, but totally up to you if you want to do that.

**Review by abrennan89 — COMMENTED** [review]
Some minor feedback. Also since this is my first week on the review squad I will have someone else double check. Thanks!

**Comment by jboxman-rh** [comment]
@JoeAldinger, so this pertains to the migration itself. But that's optional, or rather someone might already be running OVN-Kubernetes. Instead, this content belongs with the installation topics, probably, rather than here. We might already mention that it can only be configured at installation time elsewhere.

Although I see the bug specifically asks for this. And I can see some value in mentioning this possibly.

@pliurh, is it possible to set hybrid networking as part of the migration? Or is this unsupported?

File: modules/nw-ovn-kubernetes-migration.adoc (hunk 1 above)
Comment (jboxman-rh) [reply modules/nw-ovn-kubernetes-migration.adoc:105]: @JoeAldinger 

We're going to rebrand this for 4.12+, but in the meantime, I'd say

> You must configure hybrid networking with the OVN-Kubernetes cluster network provider...

> In addition, an {product-title} cluster configured with the OVN-Kubernetes cluster network provider is a...

I think.

**Comment by JoeAldinger** [comment]
@jboxman-rh we do mention it [here](https://docs.openshift.com/container-platform/4.7/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html) in the installation. I see a use case for having it in both places.

**Comment by jboxman-rh** [comment]
> @jboxman-rh we do mention it [here](https://docs.openshift.com/container-platform/4.7/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html) in the installation. I see a use case for having it in both places.

Yes, that makes sense.

**Comment by pliurh** [comment]
> is it possible to set hybrid networking as part of the migration? Or is this unsupported?
With 'hybrid networking', do you mean a cluster with Windows workers? Only OVN-Kubernetes supports hybrid cluster, Openshift-SDN doesn't. So no, we don't support it for SDN migration.

**Comment by jboxman-rh** [comment]
@JoeAldinger, we need to omit this from the migration to OVN-Kubernetes documentation.

**Comment by JoeAldinger** [comment]
> @JoeAldinger, we need to omit this from the migration to OVN-Kubernetes documentation.

Can you clarify what you mean by "this"? Do you mean that we need to say that hybrid cluster is only supported on OVN-Kubernetes and not OpenShift SDN?

**Comment by jboxman-rh** [comment]
@JoeAldinger, I mean the changes do not belong in `[modules/nw-ovn-kubernetes-migration.adoc](https://github.com/openshift/openshift-docs/pull/49449/files#diff-3ac67a3809976663e6cb55fc9e73bedbc5cb29f68f7a5967f209f26acc4d0a37)` at all, because this covers a post-installation user migration to OVN-Kubernetes from OpenShift SDN, and the latter does not support Windows nodes, so the content in this context makes no sense.

**Comment by JoeAldinger** [comment]
@jboxman ok, I moved the note to [here](https://docs.openshift.com/container-platform/4.7/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html).

**Comment by anuragthehatter** [comment]
Late to the party due to PTO. Final changes looks good to me. Thanks Joe

/lgtm

**Pull request merged**